spotlight-client/auth0/hooks/02-add-state-code-to-app-metadata-and-send-to-idtoken.js

```diff
@@ -0,0 +1,60 @@
+/**
+ * Handler that will be called during the execution of a PostLogin flow.
+ *
+ * @param {Event} event - Details about the user and the context in which they are logging in.
+ * @param {PostLoginAPI} api - Interface whose methods can be used to change the behavior of the login.
+ */
+exports.onExecutePostLogin = async (event, api) => {
```

Comment:
This looks great! One thing to consider here is performance. This logic will run every time a user logs in since it is `onExecutePostLogin`. There is not a log of logic here and none of it is iterating through huge objects so I don't think it is a problem, but another option would have been to add the state_code to the user's metadata post registration (so it will only be done once in the life of that user) then just add the metadata to the token post login. Both work, just some tradeoffs to keep in mind for next time!

Reply:
Yeah I guess looking at the user list in Auth0, we currently don't have any pre-existing users that need backfilling, so I should just move this to a post registration hook, no need to run this post login?

Reply:
Moved to post-registration action!